tests/unit/test_utils.py

```diff
@@ -261,3 +273,15 @@ def test_get_journal_and_section_invalid():
 
     assert journal_title == ''
     assert section == ''
+
+
+def test_strict_kwargs_pass(dummy_strict_kwargs_cls):
+    """Test the `strict_kwargs` decorator allowing the kwargs."""
+    dummy = dummy_strict_kwargs_cls(a=1, b=2, d=None, _x=4, settings={'DUMMY': True})
+    assert callable(dummy.__init__)
+
+
+def test_strict_kwargs_fail(dummy_strict_kwargs_cls):
+    """Test the `strict_kwargs` decorator disallowing some kwargs."""
+    with pytest.raises(TypeError):
+        dummy_strict_kwargs_cls(a=1, b=2, e=4)
```

Comment:
you could make this more readable by using better named vars:
```python
dummy_strict_kwargs_cls(goodparam1=1, goodparam2=2, badparam=4)
```
As in the test itself you can't really tell what's the action that it causes the error without having to go to the class definition, with the renaming, `badparam` really pops out.